### ISSUE TITLE
New producers and filters for the Phase 2 Muon HLT - Pending comments (#32699 and #32700)

### ISSUE THREAD

**Issue by silviodonato** [cms-sw/cmssw#32709]
@khaosmos93  After #32699 and #32700, please implement/reply to the comments on #32699 and backport them to 11_1_X.

**Comment by silviodonato** [comment]
assign hlt, reconstruction

**Comment by khaosmos93** [comment]
New PR #32713 has been made.

**Comment by slava77** [comment]
@perrotta @jpata 
was there anything left for reco for this issue?

**Comment by jpata** [comment]
@cmsbuild please close

thanks for the reminder, the fixes were implemented in https://github.com/cms-sw/cmssw/pull/32713 and backported in https://github.com/cms-sw/cmssw/pull/32751.
